mmhuman3d/data/datasets/pipelines/loading.py

```diff
@@ -43,8 +44,19 @@ def __call__(self, results):
         else:
             filename = results['image_path']
 
-        img_bytes = self.file_client.get(filename)
-        img = mmcv.imfrombytes(img_bytes, flag=self.color_type)
+        if filename.endswith('smc'):
+            assert 'image_id' in results, 'Load image from .smc, ' \
+                                          'but image_id is not provided.'
+            device, device_id, frame_id = results['image_id']
+            smc_reader = SMCReader(filename)
+            img = smc_reader.get_color(
+                device, device_id, frame_id, disable_tqdm=True)
+            img = img.squeeze()  # (1, H, W, 3) -> (H, W, 3)
+            del smc_reader
+        else:
+            img_bytes = self.file_client.get(filename)
+            img = mmcv.imfrombytes(img_bytes, flag=self.color_type)
```

Comment:
The default `channel_order` is `bgr` for [`mmcv.imfrombytes()` ](https://mmcv.readthedocs.io/en/latest/_modules/mmcv/image/io.html#imfrombytes), while `smc_reader.get_color()` returns `rgb` image, will this cause some potential problem?

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/76/commits/b518ac091630c97fafe8a045ef2b76566f8342af.